src/main/java/uk/gov/ons/census/casesvc/model/repository/CaseRepository.java

```diff
@@ -1,7 +1,10 @@
 package uk.gov.ons.census.casesvc.model.repository;
 
+import java.util.Optional;
 import java.util.UUID;
 import org.springframework.data.jpa.repository.JpaRepository;
 import uk.gov.ons.census.casesvc.model.entity.Case;
 
-public interface CaseRepository extends JpaRepository<Case, UUID> {}
+public interface CaseRepository extends JpaRepository<Case, UUID> {
+  Optional<Case> findByCaseId(UUID caseId);
```

Comment:
is it really necessary to wrap the result in an `Optional`?

Reply:
My understanding is that since Java 8 you should wrap a potential Null in an optional.  This is used with 'free' JPA functions like findById, so also feels more consistent.